Sources/SkipUI/SkipUI/Environment/EnvironmentValues.swift

```diff
@@ -282,6 +282,11 @@ extension EnvironmentValues {
         set { setBuiltinValue(key: "lineLimit", value: newValue, defaultValue: { nil }) }
     }
 
+    public var lineLimitReservesSpace: Bool? {
```

Comment:
This is not a public EnvironmentValues key in SwiftUI (correct me if I'm wrong), so it should go below in the "Internal values" section and be underscore-prefixed